cmd/faucet

```diff
@@ -19,6 +19,7 @@ source misc/config_base.sh
 
 if [ "$1" == "local" ]; then
     docker_volumes="-v $ROOT/faucet/faucet:/usr/lib/python3.6/site-packages/faucet"
+    docker_volumes="-v $ROOT/faucet/faucet:/usr/lib/python3.7/site-packages/faucet"
```

Comment:
The second line appears to overwrite the first line. Should we delete the first one?

Reply:
good catch, it should be += with a space in there somewhere

Reply:
There are some cases in which the daq/faucet image get's installed to python3.7 which overrides the "local" flag's function since local code is copied over to python3.6. this line just makes sure "local" functionality is honored no matter what.

Reply:
oh yeah, my bad. changing that.

Reply:
Changed. PTAL.